docs/diagnostics/PX1006.md

```diff
@@ -13,9 +13,11 @@ For inherited DAC types, the order of data view declarations affects the number
  - Two `PXCache` instances are created if you declare the data views so that the first data view uses the base DAC, and the second data view uses the derived type. 
  - A shared `PXCache` instance for the base DAC and the derived DAC is created if you declare the data views so that the first data view uses the derived DAC, and the second data view uses the base DAC.
 
- To use separate `PXCache` instances, you change the order of data view declarations. If you are using a shared `PXCache` instance, make sure the `PXCache` instance refers to the needed DAC type.
+The diagnostic works for graphs and graph extensions.
 
-## Example of Code That Results in the Warning
+To use separate `PXCache` instances, you change the order of data view declarations. If you are using a shared `PXCache` instance, make sure the `PXCache` instance refers to the needed DAC type.
+
+## Example of Code That Results in the Message
```

Comment:
@k-s-popova maybe we can add the diagram from our wiki:
https://wiki.acumatica.com/display/TND/Relationship+Between+.Net+Inheritance+and+the+Number+of+Caches+in+PXGraph

Reply:
@SENya1990 , I do not think that the diagnostic description is a correct place for this diagram. I would add it into documentation if needed. You've said that the behavior has been changed in 2018 R2. For which version is this diagram?

Reply:
I've made it for 2018R1 and it should be correct for earlier versions. The cache initialization has changed in 2018R2 so I don't know how it works with .Net inheritance now.

Reply:
@k-s-popova  maybe Alexander Ignatin or someone else from the Platform team knows more

Reply:
we do not perform any changes (except critical) in the documentation of 2018R1, therefore we will not add the diagram in the documentation.